src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -9,6 +9,15 @@
 report_template = """
 {dev:<25} {size:<12} {rot!s:<7} {available!s:<9} {model}"""
 
+ceph_disk_parttypes = {
```

Comment:
even though these seemed to be used here only, the ceph-disk GUIDs are in `util/constants.py` so perhaps getting these over there might be nice for consistency

Reply:
Ah thx, yes I'll use those. Two questions though:

1. The plain type has `'encrypted': True, 'encryption_type': 'plain'`. Is that intentional or should that be `'encrypted': False, 'encryption_type': 'plain'`?
2. The lockbox label is missing. Probably no harm in adding that so we can detect it in inventory right?

Reply:
1) Great question. This is part of the types of encryption dmcrypt offers, one of them is `plain` (see [some docs about this here](https://wiki.archlinux.org/index.php/Dm-crypt/Device_encryption#Encrypting_devices_with_plain_mode)) and the other one is `luks`. So `plain` *is* a valid form of encryption.

2) Yes, lets add that, might as well try to get those in... who knows if we might need them later. Good catch!

Reply:
K, this should do the trick now.